examples/test-player.rs

```diff
@@ -0,0 +1,465 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+extern crate gecko_media;
+
+use gecko_media::{GeckoMedia, Metadata, PlayerEventSink};
+use gecko_media::{PlanarYCbCrImage, Plane, TimeStamp};
+use std::env;
+use std::ffi::CString;
+use std::fs::File;
+use std::io::prelude::*;
+use std::mem;
+use std::ops::Range;
+use std::path::Path;
+use std::sync::mpsc;
+use std::thread;
+
+extern crate gleam;
+extern crate glutin;
+extern crate time;
+extern crate webrender;
+
+use webrender::api::*;
+use webrender::api::ImageData::*;
+
+mod ui;
+
+static mut CLOSE_WINDOW: bool = false;
```

Comment:
Seems like we should be able to get rid of this global (I haven't read through the whole example yet)

Reply:
Yes, done.